hivemq-edge/src/frontend/src/hooks/usePrivateMqttClient/usePrivateMqttClient.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'vitest'
+import { renderHook } from '@testing-library/react'
+
+import { SimpleWrapper as wrapper } from '@/__test-utils__/hooks/SimpleWrapper.tsx'
+import { usePrivateMqttClient } from '@/hooks/usePrivateMqttClient/usePrivateMqttClient.ts'
+import { PrivateMqttClientType } from '@/hooks/usePrivateMqttClient/type.ts'
+
+describe('usePrivateMqttClient', () => {
+  it('should load the data', async () => {
+    const { result } = renderHook(() => usePrivateMqttClient(), { wrapper })
+
+    expect(result.current).toStrictEqual<PrivateMqttClientType>({ state: undefined, actions: undefined })
```

Comment:
question: is it right that the data is empty? Does not match the title of the test